content/blog/2024/Queen_Odds_Match.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "LeelaQueenOdds vs Respectful_Dave"
+author: "Naphthalin"
+published: 2024-11-17
+---
+
+We're excited to announce a queen odds exhibition bullet match between Leela and chess streamer [CM David Maycock](https://ratings.fide.com/profile/5142547).
```

Comment:
Here it is `CM David Maycock`, below `CM Respectful_Dave`.
Maybe mention Respectful_Dave here as well 
`(aka Respectful_Dave)`

Reply:
Added the info in, linking it to his youtube profile (so it's linked twice, but that makes sense I guess).